### DISCUSSION TITLE
How to access/select the punctual lights on my glb file?

### DISCUSSION THREAD

**Discussion by gabrielgaldinomagalhaes** [google/model-viewer#4147] [Q&A]
I have a model that consists in: a black box and a punctual light inside of it. I can see the light "lightning" the box, and I can see it getting dark if I change the exposure of the model-viewer. But I wanted to now how can I selected the actual light, and for example work with the intensity or even the color of this source of light. Is it possible? 

In the video bellow I'm actually changing the exposure values of the model-viewer, but I wanted to reach the light sources. Is it possible ?

In my case use I would have more objects, not just a black box. I tried also with mesh and emissive materials, but it feels like the emissive don't really work too, because once I get the exposure down, the emissive also gets black.

https://user-images.githubusercontent.com/45290323/222087993-2ae2ce52-2118-4e99-9a49-bb29176f5259.mp4

**Comment by elalish** [comment]
We don't expose lights in our public API. You can hack down to our three.js layer by accessing our private `$scene` symbol, but at that point you're on your own and you'll have to be careful of unannounced breaking changes in future versions.

**Comment by princeku07** [comment]
@gabrielgaldinomagalhaes  have you solved this problem
